script.js

```diff
@@ -0,0 +1,101 @@
+const submit = document.getElementById('submit');
+
+const myBook = [];
+
+function Books(title, author) {
+  this.title = title;
+  this.author = author;
+}
+
+function addBook(newBook) {
+  myBook.push(newBook);
+}
+
+function displayBook(newBook) {
+  const list = document.querySelector('#book-list');
+  const row = document.createElement('div');
+  row.innerHTML = `
+    <h4>${newBook.title}</h4>
+    <h4>${newBook.author}</h4>
+    <button><a href="#" class="btn btn-danger btn-sm remove">Remove</a></button><hr>`;
+
+  list.appendChild(row);
```

Comment:
Good job 👍🏽 

- [OPTIONAL] 
the use of `DOMContentLoaded` over `load` "A different event, load, should be used only to detect a fully-loaded page. It is a common mistake to use load where DOMContentLoaded would be more appropriate.", [see](https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event )